src/index.php

```diff
@@ -29,6 +29,8 @@
         break;
       case '4':
         $message = "<div class='alert alert-warning'>You need to be logged in to view this! Please log in again.</div>";
+      case '':
```

Comment:
Why the case here is empty?
As there is no break statement after the other case the OTP error message always will pop up when someone gets logged out. 
As the login procedure currently doesn't report why the login failed, maybe just include this error message into the failed login attempt (case 3).